pkg/api/universal/subtlecrypto_subtlecrypto.go

```diff
@@ -233,7 +233,8 @@ func (a *Universal) SubtleWrapKeyAlpha1(ctx context.Context, in *runtimev1pb.Sub
 		// We are not going to return the exact error from the component to the user, because an error that is too specific could allow for various side channel attacks (e.g. AES-CBC and padding oracle attacks)
 		// We will log the full error as a debug log, but only return a generic one to the user
 		a.logger.Debug(messages.ErrCryptoOperation.WithFormat(err))
-		err = messages.ErrCryptoOperation.WithFormat("failed to wrap key")
```

Comment:
Very important: please do not return the exact error returned by the component! That can be logged, but must not be returned to the user. The old code was overriding the error for a reason.

See the comment above. The exact error must **never** be sent to users, or really bad things (side channel attacks) could happen

Reply:
Thanks for this. I should have read the comments properly. Modified the changes accordingly